src/loaders/resourceLoader.test.ts

```diff
@@ -372,10 +372,6 @@ describe("ResourceLoader::deleteSchemaVersion()", () => {
     const getSidecarStub: sinon.SinonStub = sandbox.stub(sidecar, "getSidecar");
 
     getSidecarStub.resolves(mockHandle);
-
-    getSchemaRegistryForEnvironmentIdStub = sandbox
```

Comment:
Call to getSchemaRegistryForEnvironmentId() no longer made due to being able to make use of info in the Subject object.